src/test/deploy/functions/runtimes/node/parseRuntimeAndValidateSDK.spec.ts

```diff
@@ -5,27 +5,16 @@ import * as sinon from "sinon";
 // eslint-disable-next-line
 const cjson = require("cjson");
 
-import { FirebaseError } from "../../../error";
-import * as checkFirebaseSDKVersion from "../../../checkFirebaseSDKVersion";
-import * as runtime from "../../../deploy/functions/parseRuntimeAndValidateSDK";
-import * as utils from "../../../utils";
-
-describe("getHumanFriendlyRuntimeName", () => {
-  it("should properly convert raw runtime to human friendly runtime", () => {
-    expect(runtime.getHumanFriendlyRuntimeName("nodejs6")).to.contain("Node.js");
-  });
-});
+import { FirebaseError } from "../../../../../error";
+import * as runtime from "../../../../../deploy/functions/runtimes/node/parseRuntimeAndValidateSDK";
 
 describe("getRuntimeChoice", () => {
   const sandbox = sinon.createSandbox();
   let cjsonStub: sinon.SinonStub;
-  let warningSpy: sinon.SinonSpy;
   let SDKVersionStub: sinon.SinonStub;
 
   beforeEach(() => {
     cjsonStub = sandbox.stub(cjson, "load");
-    warningSpy = sandbox.spy(utils, "logWarning");
```

Comment:
These parts of tests are moved to /test/deploy/functions/runtimes/node/versioning.spec.ts